docs/spark-getting-started.md

```diff
@@ -57,8 +57,6 @@ This command creates a path-based catalog named `local` for tables under `$PWD/w
 ```sh
 spark-sql --packages org.apache.iceberg:iceberg-spark-runtime-3.2_2.12:{{% icebergVersion %}}\
     --conf spark.sql.extensions=org.apache.iceberg.spark.extensions.IcebergSparkSessionExtensions \
-    --conf spark.sql.catalog.spark_catalog=org.apache.iceberg.spark.SparkSessionCatalog \
```

Comment:
This makes line 55 Incorrect

Reply:
Do you mean `adds support for Iceberg tables to Spark's built-in catalog` was achieved by adding a `SparkSessionCatalog`?

But then it is of type hive and URI is not mentioned. Which may confuse the users.
Also `spark_catalog` was not used in any example snippets. 

So, do you have any suggestions here?

Reply:
which one can be better?
a) Remove "built-in catalog" mentioned from the line 55 
b) or change  `spark_catalog` type also to `Hadoop`
c) or add some hive URI = xxxxxx in the example snippet 

I don't think we can leave it as it is because the catalog-type is hive but without URI will just confuse the users.

Reply:
"spark_catalog" is the built in catalog for Spark. The one you get without specifying anything else. This configuration would work for any user who has already defined a HMS for Spark and would like it to also work with Iceberg Tables.

I think it's fine to skip this usage though since all the examples use a separate "SparkCatalog"

In this case I think we should probably add some more details SparkSessionCatalog in some different section perhaps?

Reply:
https://iceberg.apache.org/docs/latest/spark-configuration/#catalog-configuration already talks about the `SparkSessionCatalog`.  So, we can remove it from here?

Reply:
I would rather leave it here. This is useful to anyone starting with an existing Spark deployment.